### PR TITLE
Coats, the Drippening

### PR DESCRIPTION
**Pull request by 4hands44**
## About The Pull Request

Adds:

Blue & Black Greatcoats for Officers
(All department heads, CO+, CIC Staff.)
Blue: https://gyazo.com/9c1fa02c1a5372308affb7357d541df2
Black: https://gyazo.com/0aebf34cbab9b29ffc3eb6cf9fbfa1af

New ww2 style tanker jacket to compete with the CO Bomber jacket. VC and IOs spawn with them out of Cryo. Features slightly higher blast resistance to represent protection from spalling and other vehicular hazards.
Closed https://gyazo.com/e6a296a61a4da0056c661106b8720ab4
Open https://gyazo.com/f0c7f31241f548a992a9c61ea87f6586

## Why it's good for the game
Drip Go BRRR

## Changelog

:cl:
qol: Replaces some instances of "Captain" In CO Attire names/descriptions
add: Black Greatcoats for Officers to the Dress Vendor (All department heads, CO+, CIC Staff.)
add: Blue ver. of above for CO+ and admins only.
add: New ww2 style tanker jacket to compete with the CO Bomber jacket. VC and IOs spawn with them out of Cryo. Features slightly higher blast resistance to represent protection from spalling and other vehicular hazards. Does not appear in any vendor.
imageadd: New sprites
code: See above

/:cl:
COATS

New ww2 style tanker jacket to compete with the CO Bomber jacket. VC and IOs spawn with them out of Cryo. Features slightly higher blast resistance to represent protection from spalling and other vehicular hazards.

Blue & Black Greatcoats for Officers
(All department heads, CO+, CIC Staff.)

### CODE REVIEW

**Review by Agameofscones — APPROVED** [review]
sprites r gud

File: code/game/machinery/vending/vendor_types/crew/commanding_officer.dm [hunk 1]
```diff
@@ -67,6 +68,7 @@ GLOBAL_LIST_INIT(cm_vending_clothing_commanding_officer, list(
 		list("STANDARD EQUIPMENT (TAKE ALL)", 0, null, null, null),
 		list("Headset", 0, /obj/item/device/radio/headset/almayer/mcom/cdrcom, MARINE_CAN_BUY_EAR, VENDOR_ITEM_MANDATORY),
 		list("Satchel", 0, /obj/item/storage/backpack/satchel/lockable, MARINE_CAN_BUY_BACKPACK, VENDOR_ITEM_MANDATORY),
+		list("Logistics IMP Backpack", 0, /obj/item/storage/backpack/marine/satchel/big, MARINE_CAN_BUY_BACKPACK, VENDOR_ITEM_MANDATORY),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/commanding_officer.dm:71]: Remove.

**Review by stanalbatross — CHANGES_REQUESTED** [review]

File: code/game/machinery/vending/vendor_types/crew/commanding_officer.dm (hunk 1 above)
Comment (4hands44) [reply code/game/machinery/vending/vendor_types/crew/commanding_officer.dm]: Bag gone

May I ask why though?

**Review by stanalbatross — APPROVED** [review]
code is fine but needs balance review

the nvgs in the co vendor may be an issue

**Comment by AmoryBlaine** [comment]
Neither piece really fits with the aesthetic of the USCM. Also why's the bomber a completely different shade when toggled open?

**Comment by morrowwolf** [comment]
NVGs gone, balance concerns gone.

**Review by stanalbatross — APPROVED** [review]

File: code/modules/gear_presets/uscm_medical.dm [hunk 2]
```diff
@@ -34,6 +34,7 @@
 	dress_hat = list()
 	dress_gloves = list(/obj/item/clothing/gloves/marine/dress)
 	dress_shoes = list(/obj/item/clothing/shoes/laceup)
+	dress_extra = list(/obj/item/storage/large_holster/ceremonial_sword/full, /obj/item/clothing/suit/storage/jacket/marine/dress/bridge_coat_grey)
```
Comment (stanalbatross) [inline code/modules/gear_presets/uscm_medical.dm:37]: why are you also giving heads of department ceremonial swords?

Comment (4hands44) [reply code/modules/gear_presets/uscm_medical.dm:37]: Whoops. I just coded the XO and Heads first. forgot to not copy the XO's sword

Comment (4hands44) [reply code/modules/gear_presets/uscm_medical.dm:37]: should be fixed now tho

File: code/game/machinery/vending/vendor_types/crew/commanding_officer.dm [hunk 3]
```diff
@@ -71,6 +71,7 @@ GLOBAL_LIST_INIT(cm_vending_clothing_commanding_officer, list(
 
 		list("COMBAT EQUIPMENT (TAKE ALL)", 0, null, null, null),
 		list("Commanding Officer's M3 Armor", 0, /obj/item/clothing/suit/storage/marine/MP/CO, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
+		list("Commanding Officer's M3 Armor", 0, /obj/item/clothing/suit/storage/marine/MP/CO/jacket, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
```
Comment (morrowwolf) [inline code/game/machinery/vending/vendor_types/crew/commanding_officer.dm:74]: Give different name in vendor for new armor

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/game/machinery/vending/vendor_types/crew/commanding_officer.dm (hunk 3 above)
Comment (4hands44) [reply code/game/machinery/vending/vendor_types/crew/commanding_officer.dm]: Fixed

**Comment by 4hands44** [comment]
merge conflict should be fixed now

**Review by stanalbatross — APPROVED** [review]

**Pull request merged**